docs/course_materials/compound_data_types.html

```diff
@@ -279,78 +592,98 @@ <h1><b>
                 <section>
                     <h3><b>Set</b></h3>
                     <ul>
-                        <li>An unordered collection of unique elements (no duplicates)</li>
-                        <li>Empty set: set()</li>
-                        <li>{1, 'blah', 5, -1}</li>
+                        <li>An unordered collection of elemenets</li>
+                        <li>Elements are unique (no duplicates)</li>
+                        <li>Mutable: Elements can change</li>
+                        <li>Unindexed: Elements can not be accessed by an index</li>
                     </ul>
                     <pre><code data-trim data-noescape class="hljs lang-python">
-s = set()
-type(s) #  &ltclass 'set'&gt
+                        numbers = {1, 2, 3}
+                        letters = {'f', 'o', 'd'}
```

Comment:
maybe add another 'o'

Reply:
It's a set, you can't have duplicates ;)

Reply:
nice one :P